tests/flags.rs

```diff
@@ -87,6 +87,17 @@ fn flag_using_long() {
     assert!(m.is_present("color"));
 }
 
+#[test]
+fn flag_using_long_with_literals() {
+    use clap::ErrorKind;
+
+    let m = App::new("flag")
+        .arg(Arg::new("--rainbow 'yet another flag'"))
```

Comment:
When I meant use `Arg::new`, I meant use the arg builder methods, not use the usage strings to build the arg.

Reply:
Sorry, now it's fixed.